src/System.cc

```diff
@@ -1401,6 +1401,14 @@ void System::InsertTrackTime(double& time)
 }
 #endif
 
+// NOTE: might not be save to call while orbslam is mapping.
```

Comment:
[q] Is this note true? I hope it is safe to call, since we take the atlas mutex.

Reply:
Right, forgot to delete it.